tracing-subscriber/src/layer/mod.rs

```diff
@@ -393,6 +393,43 @@
 //! # Ok(()) }
 //! ```
 //!
+//! ## Runtime-configurable Layers
+//!
+//! Conditionally enabling/disabling layers at runtime can be challenging due to the generics
+//! used Layers implementations. However, A Layer wrapped in an `Option` is also Layers, which
+//! allows some runtime configuration:
+//!
+//! ```
+//! # // wrap this in a function so we don't actually create `debug.log` when
+//! # // running the doctests..
+//! # fn docs() -> Result<(), Box<dyn std::error::Error + 'static>> {
+//! use tracing_subscriber::{Registry, prelude::*};
+//! use std::{fs::File, sync::Arc};
```

Comment:
`PathBuf` needs to be imported btw
```suggestion
//! use std::{fs::File, sync::Arc, path::PathBuf};
```